SleepEngine/Engine/Render/UpdateRenderBridge.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <Engine/Render/DrawCall.h>
+
+BEGIN_NAMESPACE_SLEEP
+
+// stores render data
+// syncs it between update and render
+class UpdateRenderBridge
+{
+public:
+    std::vector<DrawCall> const& getUpdatedData() { return m_updatedData; }
+    std::vector<DrawCall> const& getLastUpdatedData() { return m_lastUpdatedData; }
+    std::vector<DrawCall>& getRenderedData() { return m_renderedData; }
```

Comment:
CONST_REF_GETTER?

Reply:
and REF_GETTER

Reply:
same thing as with previous getters, we need to change them